### PR TITLE
OSOE-682: Upgrading Lombiq.UITestingToolbox to Orchard Core 1.7

### PR DESCRIPTION
**Pull request by Psichorex**
[OSOE-682](https://lombiq.atlassian.net/browse/OSOE-682)

[OSOE-682]: https://lombiq.atlassian.net/browse/OSOE-682?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Lombiq.Tests.UI.Shortcuts/Lombiq.Tests.UI.Shortcuts.csproj
Comment (Piedone) [inline Lombiq.Tests.UI.Shortcuts/Lombiq.Tests.UI.Shortcuts.csproj]: You were supposed do publish NuGet alphas after all PRs are approved, since if there's and review feedback, you'll need to do a release again. Hopefully, there won't be.

File: Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs
Comment (Piedone) [inline Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs:82]: Why is this change necessary? `LoadConfiguration()` looks like it still does the same thing.

**Review by Piedone — CHANGES_REQUESTED** [review]
Also remove obsolete members from `SeleniumLogEntryExtensions` and `AtataFactory`.

File: Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs
Comment (Psichorex) [reply Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs:82]: I had a deprecation warning on `LoadConfiguration` suggesting to use `LoadConfigurationFromFile()` instead.

File: Lombiq.Tests.UI.Shortcuts/Lombiq.Tests.UI.Shortcuts.csproj
Comment (Psichorex) [reply Lombiq.Tests.UI.Shortcuts/Lombiq.Tests.UI.Shortcuts.csproj]: I only did alpha releases for those projects that are being dependent on by other projects to see if there would be any issue.

Comment (Piedone) [reply Lombiq.Tests.UI.Shortcuts/Lombiq.Tests.UI.Shortcuts.csproj]: Without fully upgrading the NuGet solution you won't be able to see any issues.

File: Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs
Comment (Piedone) [reply Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs:82]: I see, good then.

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]

**Pull request merged**
